relaxation_procedure.py

```diff
@@ -24,12 +24,12 @@ def relaxation_procedure():
 	weight = [1,1,1]
 	eta = 0.3
 
-	#for i in xrange(len(dataset)):
-	#	mod_value.append(square_values(dataset[i])
```

Comment:
these were some functions initially and later removed,
So either delete them or try to understand the syntax of the code please.

Here it was initially a for loop and in python you need to have an indentation I guess.